templates/nginx/galaxy-main.j2

```diff
@@ -77,6 +77,10 @@ server {
 		add_header X-Frame-Options SAMEORIGIN;
 		add_header X-Content-Type-Options nosniff;
 
+		# Add upstream response headers that would otherwise be omitted
+		add_header Access-Control-Allow-Origin $upstream_http_access_control_allow_origin;
+		add_header Access-Control-Allow-Methods $upstream_http_access_control_allow_methods;
+
 		# CORS
                 if ($http_origin ~ "^https?://(covid19map\.elixir-luxembourg\.org|avivator\.gehlenborglab\.org|www\.bx\.psu\.edu)$") {
```

Comment:
https://github.com/galaxyproject/usegalaxy-playbook/pull/386/files#r1538948168

Reply:
Thank you! I will convert this PR to a draft and wait for the conclusion

Reply:
Thanks @sanjaysrikakulam, appreciate you taking care of this!